<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAxialAnalysisOptionsDlg</class>
 <widget class="QDialog" name="CAxialAnalysisOptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Axial Analysis Options</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Radius / list of radii, e.g., 2,3,n</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="c_radius"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="c_choice">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Include choice (betweenness)</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c_local">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Include local measures</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c_rra">
       <property name="text">
        <string>Include RA, RRA and total depth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="c_weighted">
       <property name="text">
        <string>Include weighted measures</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Weight by</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_attribute_chooser"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="c_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>c_radius</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CAxialAnalysisOptionsDlg</receiver>
   <slot>OnUpdateRadius()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_weighted</sender>
   <signal>clicked()</signal>
   <receiver>CAxialAnalysisOptionsDlg</receiver>
   <slot>OnWeighted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_ok</sender>
   <signal>clicked()</signal>
   <receiver>CAxialAnalysisOptionsDlg</receiver>
   <slot>OnOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>c_cancel</sender>
   <signal>clicked()</signal>
   <receiver>CAxialAnalysisOptionsDlg</receiver>
   <slot>OnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnUpdateRadius()</slot>
  <slot>OnWeighted()</slot>
  <slot>OnOK()</slot>
  <slot>OnCancel()</slot>
 </slots>
</ui>
